cmd/harmony/main.go

```diff
@@ -253,9 +253,9 @@ func createGlobalConfig() *nodeconfig.ConfigType {
 	consensusPriKey := &bls.SecretKey{}
 
 	if *isGenesis {
-		err := consensusPriKey.DeserializeHexStr(genesisAccount.BlsPriKey)
+		err := consensusPriKey.DeserializeHexStr(genesisAccount.DummyKey)
```

Comment:
why using this DymmyKey? BlsPriKey is also generated keys by us currently?

Reply:
To avoid people's confusion.